content/posts/transformer-part1/index.md

```diff
@@ -140,68 +132,69 @@ class TensorView {
 };
 
 ```
-The Chapel version, however, uses an alternative approach. It uses built-in arrays to represent all matrices and tensors. Unlike C++, `ref` is not allowed to be in `class` or `record`, so a `TensorView` structure-like in Chapel cannot be constructed. I see this as a feature that would be beneficial to implement in the future.
+The Chapel version, however, uses an alternative approach. It uses built-in arrays to represent all matrices and tensors. Unlike C++, `ref` fields are not currently supported in a `class` or `record`, so a `TensorView`-like structure in Chapel cannot be constructed:
 ```Chapel
 class TensorView {
-  // ref data; error, ref can not be declare in class or record 
+  ref data;  // error: References cannot be members of classes or records yet.
```

Comment:
Here, I uncommented the field to make it render more clearly and quoted the error message from the compiler into a comment.